Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationItem.m

```diff
@@ -37,6 +37,19 @@ @interface BPKHorizontalNavigationItem ()
 
 @implementation BPKHorizontalNavigationItem
 
+- (instancetype)initWithOption:(BPKHorizontalNavigationOption *)option {
```

Comment:
Should this be `id<BPKHorizontalNavigationOptionType>`?

Reply:
Strictly speaking yes but this whole class doens't work if it's not `BPKHorizontalNavigationOption` so it should probably throw